yeonjae/baekjoon/브루트포스/7568-덩치.js

```diff
@@ -0,0 +1,36 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  input.shift();
+
+  const physicalList = input.map((el) => el.split(" ").map(Number));
+  const result = [];
+
+  for (let i = 0; i < physicalList.length; i++) {
+    let grade = 1;
+    for (let j = 0; j < physicalList.length; j++) {
+      if (i !== j) {
+        if (
+          physicalList[i][0] < physicalList[j][0] &&
+          physicalList[i][1] < physicalList[j][1]
+        )
+          grade++;
+      }
+    }
+    result.push(grade);
+  }
+
+  console.log(result.join(" "));
```

Comment:
```suggestion
  const n = +input.shift();
  const physicalList = input.map((el) => el.split(" ").map(Number));
  const result = [];

  for (let i = 0; i < n; i++) {
    let grade = 1;
    
    for (let j = 0; j < n; j++) {
      if (i === j) continue;
      if (
        physicalList[i][0] < physicalList[j][0] &&
        physicalList[i][1] < physicalList[j][1]
      ) {
        grade++;
      }
    }
   
    result.push(grade);
  }

  console.log(result.join(" "));
```

- 풀이 잘봤습니다 🙇‍♂️
- shift의 반환값이 n이기 때문에 반환되는 값을 변수에 할당해서 이를 활용하면 어떨까 싶네요 
- 그리고 줄바꿈이 되는 경우에는 무조건 중괄호 써주셔야합니다!!기억해주세요!! (grade++)부분 
- 또한, 문단 별로 잘 나눠주세요 (변수 선언부, 로직부)

Reply:
넵 중괄호에대한부분을 계속 놓치고있던거같아요ㅠㅠ 컨벤션 신경 더 쓰겠습니다!

Reply:
넵 중괄호에대한부분을 계속 놓치고있던거같아요ㅠㅠ 컨벤션 신경 더 쓰겠습니다!